### ISSUE TITLE
Datasets help wanted: Tight coupling of IMU and GNSS

### ISSUE THREAD

**Issue by waynehq** [weisongwen/GraphGNSSLib#8]
hi,thanks for your contribution!
I want to use Tight coupling algorithm to fuse raw imu data and gnss raw data.But the datasets does not provide extrinsic Parameters between imu and antenna.At the same time,FGO is not fuse imu. Can help?

**Comment by weisongwen** [comment]
Hi, thank you for your interest. you may like our recently opensourced Urbannav dataset here https://github.com/weisongwen/UrbanNavDataset. The dataset provide the GNSS and IMU data. 

BTW, the GraphGNSSLib does not provide the GNSS INS integration.

**Comment by waynehq** [comment]
Thanks for your reply!Any plan for GNSS INS integration later?

**Comment by weisongwen** [comment]
Yes! We are planning this. But now we are focusing on extending the GraphGNSSLib to multi-frequencies.

**Comment by weisongwen** [comment]
Feel free to open if needed
